shell/platform/linux/fl_key_event_plugin.cc

```diff
@@ -20,11 +24,113 @@ static constexpr char kUnicodeScalarValuesKey[] = "unicodeScalarValues";
 static constexpr char kGtkToolkit[] = "gtk";
 static constexpr char kLinuxKeymap[] = "linux";
 
+static constexpr uint64_t kMaxPendingEvents = 1000;
+
+// Declare and define a private pair object to bind the id and the event
+// together.
+
+G_BEGIN_DECLS
```

Comment:
Are these required in a C file? [docs](https://developer.gnome.org/glib/stable/glib-Miscellaneous-Macros.html#G-BEGIN-DECLS:CAPS).